week4/homework-template/.gitignore

```diff
@@ -1,46 +1,46 @@
-# Logs
-*.log
-npm-debug.log*
-yarn-debug.log*
-yarn-error.log*
-
-# Runtime data
-pids
-*.pid
-*.seed
-*.pid.lock
-
-# Directory for instrumented libs generated by jscoverage/JSCover
-lib-cov
-
-# Coverage directory used by tools like istanbul
-coverage
-
-# nyc test coverage
-.nyc_output
-
-# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
-.grunt
-
-# Bower dependency directory (https://bower.io/)
-bower_components
-
-# Dependency directories
-node_modules/
-jspm_packages/
-
-# dotenv environment variables file
-.env
-
-# add gitignore for .vscode and .vscode/settings.json
-.vscode 
-.vscode/settings.json
-
-# React build
-build
-
-dist/
-
-.DS_Store
-
+# Logs

```

Comment:
It apparently thinks these files below (which are all almost exactly the same) are completely new. Has something to do with unix line endings, I suspect.